compiler/damlc/lib/DA/Cli/Damlc/Test.hs

```diff
@@ -86,6 +94,40 @@ failedTestOutput h file = do
     pure $ map (, Just errMsg) $ fromMaybe [VRScenario file "Unknown"] mbScenarioNames
 
 
+printTestCoverage :: [(VirtualResource, SS.ScenarioResult)] -> Int -> Int -> IO ()
+printTestCoverage results nrOfTemplates nrOfChoices =
+    putStrLn $
+    unwords
+        [ "test coverage: templates"
+        , percentage coveredNrOfTemplates nrOfTemplates <> ","
+        , "choices"
+        , percentage coveredNrOfChoices nrOfChoices
+        ]
```

Comment:
It would be nice (as a separate step) to add a `--coverage` flag that lists the uncovered templates or choices.

Reply:
that would be cool, I'll add it in a follow up PR.